src/python/globalfuncs.cpp

```diff
@@ -1001,6 +1162,18 @@ static PyMethodDef Essentia__Methods[] = {
   { "hz2bark",       hzToBark,         METH_O, "Converts a frequency in Hz to a bark band" },
   { "mel2hz",        melToHz,          METH_O, "Converts a mel band to frequency in Hz" },
   { "hz2mel",        hzToMel,          METH_O, "Converts a frequency in Hz to a mel band" },
+  { "midi2hz",       midiToHz,         METH_VARARGS, "Converts a midi note number to frequency in Hz" },
+  { "hz2midi",       hzToMidi,         METH_VARARGS, "Converts a frequency in Hz to a midi note number" },
+  { "hz2cents",      hzToCents,        METH_VARARGS, "Returns the cents distance between two frequencies in Hz" },
```

Comment:
if we use the `referenceFrequency` (see my comment about), we should updates this description too.